README.md

```diff
@@ -72,6 +72,12 @@ A cukinia config file supports the following statements:
 * ``cukinia_knoerror <priority>``: Validate kernel has booted without important
   errors (the priority argument is the log level number to check)
 * ``cukinia_sysctl <parameter> <value>``: Validate kernel sysctl parameter is set to value
+* ``cukinia_network_interface_up <network interface name>``: Validate that interface is up
```

Comment:
Use <interface> as the placeholder, other placeholders either don't have spaces or uses underscores.

Reply:
Resolved